airflow/gcp/hooks/gcs.py

```diff
@@ -238,7 +260,6 @@ def exists(self, bucket_name, object_name):
     def is_updated_after(self, bucket_name, object_name, ts):
         """
         Checks if an blob_name is updated in Google Cloud Storage.
-
```

Comment:
This change prevents the documentation from being built properly. Can you delete it?